Simonyi/Troubleshooting/MTCS/MTM2/M2-recovery.rst

```diff
@@ -0,0 +1,338 @@
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Kevin Fanning*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Yiyung Kang, Jacqueline Seron*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _MTM2-Troubleshooting-M2-recovery:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+M2 Recovery
+#########################
+
+.. ..  TO DO: CHECK internal links are working, and see if page on LOVE commands has a counterpart in obs-ops
+
+.. _M2-recovery-Overview:
+
+Overview
+^^^^^^^^
+.. In one or two sentences, explain when this troubleshooting procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This procedure should be done when MTM2 goes to ``FAULT`` and exits the *closed-loop* state.
+
+This step-by-step guide is the integration of the `Recovery system at night`_ and 
+`Restart control system`_ procedures
+
+.. _`Recovery system at night`:  https://ts-m2gui.lsst.io/error-handling/recover-system-night.html
+
+.. _`Restart control system`: https://ts-m2gui.lsst.io/error-handling/error-handling.html#lsst-ts-m2gui-error-restart-control-system
+
+
+
+.. admonition:: Information
+
+    **MTM2 possible states** or control modes are *Idle*, *TelemetryOnly (Disabled)*, *Open Loop*, *Closed Loop*. 
+    Each state is indicated by an integer from 1 to 4.
+
+    .. _M2-recovery-check-M2-state:
+
+    You can check the MTM2 state in Chronograf dashboards `M2 state`_, `MTM2 dashboard`_ or  `LOVE view`_, once you are signed in.
+
+    MTM2 should always be in the *closed-loop* state (state 4). 
```

Comment:
Suggestion: add "... should always be in the *closed-loop* state for the safety of the glass."

Reply:
Potentially this should be in its own ".. admonition:: Important" to really stress  the importance of recovering M2 at any time.